contrib/uncrustify/uncrustify.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+SRCROOT=`git rev-parse --show-toplevel`
+CFG="$SRCROOT/contrib/uncrustify/uncrustify.cfg"
+echo "srcroot: $SRCROOT"
+
+case "$1" in
```

Comment:
I would suggest to make a parameter mandatory for directory or file to run uncrustify on.  It would let this be scalable to more of the code base for later rather than just the goodixmoc directory.

Reply:
script updated to support folder input